requirements.txt

```diff
@@ -1,3 +1,11 @@
-Flask
-mongo
-gunicorn
+Cerberus==1.2
+click==6.7
+Eve==0.8
+Events==0.3
+Flask==1.0.2
+itsdangerous==0.24
```

Comment:
waaaat?! lol

Reply:
haha, a bad name.
But it's just a Flask requirement.

Reply:
Should we do the whole requirements.in/requirements.txt? I like that idea